pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -340,5 +340,36 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 				},
 			},
 		},
+		{
+			Type:        "discord",
+			Name:        "Discord",
+			Heading:     "Discord settings",
+			Description: "Sends notifications to Discord",
+			Options: []alerting.NotifierOption{
+				{
+					Label:        "Message Content",
+					Description:  "Mention a group using @ or a user using <@ID> when notifying in a channel",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					Placeholder:  `{{ template "default.message" . }}`,
+					PropertyName: "message",
+				},
+				{
+					Label:        "Webhook URL",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					Placeholder:  "Discord webhook URL",
+					PropertyName: "url",
+					Required:     true,
+				},
+				{ // New in 8.0.
+					Label:        "Message Description",
+					Description:  "A description added to the notification",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					PropertyName: "description",
+				},
```

Comment:
Given we have a message content above, do we required this one? I think we can keep only one of them and put it in the right place.